src/main/java/az/edu/turing/module01/lesson2/PlayerComparator.java

```diff
@@ -0,0 +1,20 @@
+package az.edu.turing.module01.lesson2;
+
+import java.util.Comparator;
+
+public class PlayerComparator implements Comparator<Player> {
+    @Override
+    public int compare(Player p1, Player p2) {
+        int scoreComparison = Integer.compare(p2.getScore(), p1.getScore());
+        if (scoreComparison != 0) {
+            return scoreComparison;
+        }
+
+        int ageComparison = Integer.compare(p1.getAge(), p2.getAge());
+        if (ageComparison != 0) {
+            return ageComparison;
+        }
+
```

Comment:
please check the condition again, I think there should be more comparisions, also try to write inline code instead creating new one, as Comparator<> is a functional interface